### DISCUSSION TITLE
Multilanguage readme

### DISCUSSION THREAD

**Discussion by sguinetti** [iptv-org/community#3] [Ideas]
Greetings. With the growth of channels in the LATAM and HISPAM lists, I was thinking of including a section to link to other README (or léame in spanish) in different languages. This makes it easier for the users when their are not English-speaking.

The files are name README-lang.md, per example README-es.md for Spanish-speakers.

**Comment by BellezaEmporium** [comment]
It would have been interesting if GitHub gave us multilingual README support.

**Comment by freearhey** [comment]
The problem is that if we translate README into at least a few major languages, the slightest change in it will require editing all the other languages as well. That will make life very difficult for all project maintainers.

Besides, the main browsers already have built-in tools for automatic translation, which already solves this problem.
